terraform/modules/origin/variables.tf

```diff
@@ -43,3 +47,22 @@ variable "public_zone_id" {
 variable "workspace" {
   type = string
 }
+
+variable "is_default_workspace" {
+  type    = bool
+  default = true
```

Comment:
I think it would be safer not to have the default here - callers of this module should always be passing this in explicitly (otherwise they won't be workspace aware).

Reply:
ok, can do